common/auth-proxy/overlays/oauth2-proxy/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: oauth2-proxy-authservice
+  labels:
+    app: authservice
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: oauth2-proxy
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app.kubernetes.io/name: oauth2-proxy
+    spec:
+      volumes:
+        - name: configmain
+          configMap:
+            name: oauth2-proxy
+            defaultMode: 420
+        - name: configalpha
+          configMap:
+            name: oauth2-proxy-alpha
+            defaultMode: 420
+        - name: authservice-client
+          secret:
+            secretName: authservice-client
+      containers:
+        - name: oauth2-proxy
+          image: quay.io/oauth2-proxy/oauth2-proxy:v7.4.0
+          args:
+            - '--alpha-config=/etc/oauth2_proxy/oauth2_proxy.yml'
+            - '--config=/etc/oauth2_proxy/oauth2_proxy.cfg'
+            - '--silence-ping-logging'
+            - '--redirect-url=http://localhost:8080/oauth2/callback'
```

Comment:
I tried, if I do, it will redirect to `https://{current-host}/oauth2/callback` it decide alone to go from http to https, pretty annoying when using port forward